### DISCUSSION TITLE
AssertionError: Expected all Python environments to have version '1', but got: 'Autosave_...'

### DISCUSSION THREAD

**Discussion by furtheraway** [microsoft/InnerEye-DeepLearning#575] [Q&A] [answered]
Dear all,

I was trying to run the HelloWorld example with Azureml

`$python InnerEye/ML/runner.py --model=HelloWorld --azureml=True`

I saw the following warning in the shell immediately after execute the above command:

`WARNING:azureml.core:Failure while loading azureml_run_type_providers. Failed to load entrypoint automl = azureml.train.automl.run:AutoMLRun._from_run_dto with exception (cloudpickle 2.0.0 (c:\miniconda\envs\innereye\lib\site-packages), Requirement.parse('cloudpickle<2.0.0,>=1.1.0'), {'azureml-dataprep'}).
Starting InnerEye runner at InnerEye/ML/runner.py`

Then the job started running in Azure ML, then failed, with the following error message:

`User program failed with AssertionError: Expected all Python environments to have version '1', but got: 'Autosave_2021-10-18T17:06:24Z_f51cd641`

I have tested one other configuration with more substantial dataset (I extended HeadAndNeckBase.py and tested it on a dataset of 5 real DICOM>Nifti scans). The Job would run for few hours on Azure GPU cluster, and then fail with the same error message.

Thank you,
John

**Comment by ktakeda1** [comment, marked as answer]
Thank you for flagging this. We have now released a fix for the issue you are seeing. Please retry your job and see if it works now.
